src/Repository/UserRepository.php

```diff
@@ -442,10 +442,16 @@ public function findBannedPaginated(
     public function findAdmin(): User
     {
         // @todo orderBy lastActivity
-        return $this->createQueryBuilder('u')
+        $result = $this->createQueryBuilder('u')
             ->andWhere("JSONB_CONTAINS(u.roles, '\"".'ROLE_ADMIN'."\"') = true")
+            ->andWhere('u.isDeleted = false')
             ->getQuery()
-            ->getResult()[0];
```

Comment:
I know I'm late but I wonder why this didn't have a limit applied to it :thinking:

Reply:
I mean how many admins will an instance have :D 
I'd say most instances are running with exactly one admin

Reply:
hmm, i wonder what happens where there's more than 1, though... it might hose whatever is expecting only 1 like assigning a magazine owner? you're probably right about there usually only being 1 admin, but these edge cases might come back to bite later.

Reply:
It's not important, I just feel like if you already know the limitations you expect, it's good to declare, so I might want to add `->setMaxResults(1)` to this or something, but also like I said not important

Reply:
> I mean how many admins will an instance have :D I'd say most instances are running with exactly one admin

Mine has three